transaction-history-manager/src/test/java/com/qpay/transactionhistorymanager/controller/TransactionHistoryControllerIntegrationTest.java

```diff
@@ -0,0 +1,74 @@
+package com.qpay.transactionhistorymanager.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.qpay.transactionhistorymanager.model.entity.TransactionEntity;
+import com.qpay.transactionhistorymanager.service.impl.TransactionHistoryServiceImpl;
+import com.qpay.transactionhistorymanager.utility.PathUtils;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.mongo.AutoConfigureDataMongo;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@WebMvcTest(controllers = TransactionHistoryController.class)
+@AutoConfigureDataMongo
+class TransactionHistoryControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private TransactionHistoryServiceImpl transactionHistoryService;
+
+    @Test
+    void should_getTransactionsByUserIdOnSelectedPage() throws Exception {
```

Comment:
```suggestion
    void should_getTransactions_when_pageProvided() throws Exception {
```

then check another case, when page is not provided and you use default values